### PR TITLE
Formatting and packaging

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by Natim** [comment]
Where do you put the python3 support now ?

**Comment by leplatrem** [comment]
Here ? :)

**Comment by Natim** [comment]
Ok why not :)

**Comment by Natim** [comment]
We can merge that to in dev then.

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
+script: tox -e $TOX_ENV
 install:
-  - make install
-  # Use colander dev version (until 1.0 released)
-  - bin/pip install --upgrade -e git+https://github.com/Pylons/colander.git#egg=colander
-script:  make tests
-after_success:
-  # Report coverage results to coveralls.io
-  - pip install coveralls
-  - coveralls
```
Comment (leplatrem) [inline .travis.yml:15]: Getting rid of coveralls was intentional ?

Comment (Natim) [inline .travis.yml:15]: No you are right, I will put it back (even if it doesn't really works well.)

File: daybed/tests/test_functional.py [hunk 2]
```diff
@@ -10,6 +11,11 @@
 
 class DaybedViewsTest(BaseWebTest):
 
+    def __init__(self, *args, **kwargs):
+        super(DaybedViewsTest, self).__init__(*args, **kwargs)
+        if not hasattr(self, 'assertCountEqual'):
+            self.assertCountEqual = self.assertItemsEqual
```
Comment (leplatrem) [inline daybed/tests/test_functional.py:17]: Are they equivalent ?

Comment (Natim) [inline daybed/tests/test_functional.py:17]: Yes they are : http://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual
I created a ticket in six to add the support for it. https://bitbucket.org/gutworth/six/issue/50/sixassertcountequal-self

**Comment by Natim** [comment]
Oh I forgot a failling test for py33 : `daybed.tests.test_functional:CityModelTest.test_data_deletion`

The question is, who is putting the location header in bytes...

**Comment by Natim** [comment]
Tox is happy, me too.
Good night guys :)

```
  py26: commands succeeded
  py27: commands succeeded
  py33: commands succeeded
  flake8: commands succeeded
  congratulations :)
```

**Pull request merged**